data/ms/legislature/Shanda-Yates-de5a9a25-1b08-45ba-9aff-e635905498ba.yml

```diff
@@ -5,21 +5,31 @@ family_name: Yates
 email: syates@house.ms.gov
 image: http://billstatus.ls.state.ms.us/members/house/yates.jpg
 party:
-- name: Democratic
+- end_date: '2022-12-29'
```

Comment:
End date should be january 2, 2024. They switched from democrat to independent on January 13, 2022